pkg/controller/integrationplatform/initialize_test.go

```diff
@@ -115,29 +115,3 @@ func TestTimeouts_Truncated(t *testing.T) {
 
 	assert.Equal(t, 5*time.Minute, answer.Status.Build.GetTimeout().Duration)
 }
-
-func TestDefaultMavenSettingsApplied(t *testing.T) {
```

Comment:
Any reason why this test was removed? I could not find an equivalent one for the new development.

Reply:
The reason is that the logic being tested is not used anymore which voids the purpose of the test. The _user_ Maven settings aren't reconciled anymore.